source/get-started.html.erb

```diff
@@ -24,7 +24,7 @@ title: Get started
         </h2>
 
         <div class="task-list-items">
-          <p> <a href="https://docs.wifi.service.gov.uk/requirements.html#requirements">Read the documentation</a> and make sure you meet all the technical requirements.</p>
+          <p>Make sure you <a href="https://docs.wifi.service.gov.uk/?_ga=2.126022537.587131065.1541080458-1055378011.1532000279#requirements">meet the technical requirements</a>.</p>
```

Comment:
Was the GA part of this link included on purpose? Wondering what it does, if so.

Reply:
No not on purpose so happy to get rid of it.

Reply:
I can use the new fancy github feature for this!

```suggestion
          <p>Make sure you <a href="https://docs.wifi.service.gov.uk/#requirements">meet the technical requirements</a>.</p>
```

Reply:
Think you should be able to accept this suggestion, then we can get it live :)